ci/vm.sh

```diff
@@ -404,6 +410,27 @@ test_provision() {
 	diff -u <(jq -cS . <<<"$want") <(jq -cS . <<<"$got")
 }
 
+test_boot_and_phone_home() {
+	run_vm &
+	local vmpid=$! i=0 t=10
```

Comment:
This variable declaration would be more readable if split across multiple statements.